charts/primary-site/templates/ingress.yaml

```diff
@@ -0,0 +1,27 @@
+{{- with .Values.ingress }}
+{{- if .enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: site
```

Comment:
Does this name supposed to be `stream` (as in the website docs)?

Reply:
I think not? Seemed to work fine being called site. This is the name of the ingress rather than the service.